code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -11,15 +11,15 @@
 #define FIRE_PLASMA_ENERGY_RELEASED			3000000	//Amount of heat released per mole of burnt plasma into the tile
 //General assmos defines.
 #define WATER_VAPOR_FREEZE					200
-#define BROWNS_FORMATION_ENERGY				100000
+#define NO2_FORMATION_ENERGY				100000
 #define TRITIUM_BURN_OXY_FACTOR				100
 #define TRITIUM_BURN_TRIT_FACTOR			10
 #define SUPER_SATURATION_THRESHOLD			96
-#define STIMULUM_HEAT_SCALE					100000
-#define STIMULUM_FIRST_RISE					0.65
-#define STIMULUM_FIRST_DROP					0.065
-#define STIMULUM_SECOND_RISE				0.0009
-#define STIMULUM_ABSOLUTE_DROP				0.00000335
+#define STIMGAS_HEAT_SCALE					100000
+#define STIMGAS_FIRST_RISE					0.65
+#define STIMGAS_FIRST_DROP					0.065
+#define STIMGAS_SECOND_RISE					0.0009
+#define STIMGAS_ABSOLUTE_DROP				0.00000335
```

Comment:
if you can't even be assed to change the full name of the variables to match up with how it used to be, why are you changing it?